engine/src/flutter/impeller/display_list/aiks_dl_text_unittests.cc

```diff
@@ -844,5 +872,31 @@ TEST_P(AiksTest, SingleIconShadowTest) {
             1u);
 }
 
+TEST_P(AiksTest, LoAndBehold) {
+  DisplayListBuilder builder;
+  DlPaint paint;
+  paint.setColor(DlColor::ARGB(1, 0.1, 0.1, 0.1));
```

Comment:
You need to scale by the content scalar too to fix the golden images.

Reply:
Why do we need to add an additional scale - the transform I'm using below is just multiplied by the ctm?

Reply:
DPI differences between CI and running locally.  The CTM is wrong for goldens on CI that run at 1px = 1 logical px.  Something is obviously weird since running goldens locally gives the wrong scale.  If we follow the convent though everything works so we haven't prioritized investigating further.